components/shared/Play/Play.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import Icon from "../Icon/Icon";
+import "./Play.module.scss";
+
+interface PlayProps {
+  onClick: () => void;
+}
+
+const Play: React.FC<PlayProps> = ({ onClick }) => {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  const handleTogglePlay = () => {
+    setIsPlaying(!isPlaying);
+    onClick();
+  };
+
+  return (
+    <button onClick={handleTogglePlay}>
+      {isPlaying ? <Icon variant="play" /> : <Icon variant="pause" />}
```

Comment:
You can simplify statement
```suggestion
      <Icon variant={isPlaying ? "play" : "pause"} />
```